.travis.yml

```diff
@@ -13,6 +13,7 @@ script:
   # tests are not done with mvn test - dont know why
   - ./scripts/build_mvn.sh
   - docker run -it --rm -v "$PWD":/opt/app-root/src -w /opt/app-root/src -u "$(id -u "${USER}"):$(id -g "${USER}")" trion/ng-cli make fintech-ui
+  - docker run -it --rm -v "$PWD":/opt/app-root/src -w /opt/app-root/src -u "$(id -u "${USER}"):$(id -g "${USER}")" trion/ng-cli-karma make consent-ui-test
```

Comment:
Technically we may end in the situation that we are testing with one toolset and compiling with other as containers are different... Maybe we can build and test in one container?

Reply:
totally makes sense. Fixed